pages/helploader.js

```diff
@@ -0,0 +1,4 @@
+import { Loader } from '../components/Loader';
+
+const HelpLoader = () => <Loader />;
+export default HelpLoader;
```

Comment:
What purpose does this serve? Do you know this will be accessible on `yoururl.com/helploader` ?